### DISCUSSION TITLE
Hide locked conversations by default

### DISCUSSION THREAD

**Discussion by PaulRBerg** [community/community#49410] [Discussions] [answered]
### Select Topic Area

Product Feedback

### Body

We love the Discussions product! We have lots and lots of discussions in our organization repositories, and we use this product for both internal and external communication.

What we don't like about it is that locked conversations are not hidden by default.

I personally would have expected the discussions tab to mirror the logic of the issues tab, which hides closed issues by default.

Will you consider hiding locked conversations by default?

**Comment by LindseyB** [comment, marked as answer]
We added the new [close discussion](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion) feature which is a bit stronger than a lock and will hide by default. Does this work for your usecase?

**Reply by PaulRBerg** [reply]
Yes, closing discussion works even better! Thanks, Lindsey.
